### ISSUE TITLE
Delineate All Bank Transaction Codes in Enum

### ISSUE THREAD

**Issue by svapnil** [svapnil/iso20022.js#15]
ISO20022 Bank Transaction Codes are currently stored as strings, but it would be great if we could translate this file into a structured enum: 

https://www.iso20022.org/sites/default/files/media/file/BTC_Codification_30October2023.xls

Maybe even include a script!

Payments / CashManagement are probably the only domains that are relevant for this package so far. That said, it does include hundreds of subfamilies that we'd have to deal with. In the future I'm imagining something like:

```
export const DomainFamily = {
    Payments: {
        ...
    },
    CashManagement: {
        AccountBalancing: "ACCB",
        ....
    }
} as const;


export const SubdomainFamily = {
    Payments: {
        ...
    },
    CashManagement: {
        AccountBalancing: {
              CrossBorder: "XBRD",
              Sweeping: "SWEP"
       }
        ....
    }
} as const;
```

Basically, we'd codify https://www.iso20022.org/catalogue-messages/additional-content-messages/external-code-sets . I'll make this an Issue so it can be perhaps tackles in the future!

_Originally posted by @svapnil in https://github.com/svapnil/iso20022.js/pull/14#discussion_r1845929507_
